WordPress/gutenberg
# Update icons

## Description

A slew of icons were still using the old vectors, not the new ones from Figma. This PR updates a bunch, and retires a few.

Retired because they were unused:

<img width="267" alt="Screenshot 2021-05-06 at 11 27 45" src="https://user-images.githubusercontent.com/1204802/117278574-2cb43380-ae61-11eb-822e-6e2632340572.png">

<img width="272" alt="Screenshot 2021-05-06 at 11 28 25" src="https://user-images.githubusercontent.com/1204802/117278587-2e7df700-ae61-11eb-837d-01009e056946.png">

<img width="271" alt="Screenshot 2021-05-06 at 10 50 11" src="https://user-images.githubusercontent.com/1204802/117278597-30e05100-ae61-11eb-901d-97d9c5e063d5.png">

Retired and swapped out: 

<img width="278" alt="Screenshot 2021-05-06 at 11 50 47" src="https://user-images.githubusercontent.com/1204802/117278864-6f760b80-ae61-11eb-8057-de1f07f25dd7.png">

_Swapped this one with "fullscreen"_.

Updated:

<img width="358" alt="Screenshot 2021-05-06 at 11 28 43" src="https://user-images.githubusercontent.com/1204802/117278903-7866dd00-ae61-11eb-8cdf-a66dba12b0f9.png">

<img width="297" alt="Screenshot 2021-05-06 at 11 39 19" src="https://user-images.githubusercontent.com/1204802/117278910-79980a00-ae61-11eb-9fb8-51d18ead43d4.png">

<img width="275" alt="Screenshot 2021-05-06 at 11 57 13" src="https://user-images.githubusercontent.com/1204802/117279659-37bb9380-ae62-11eb-8cfb-b03204e13289.png">


## To test:

Compare the icons in https://wordpress.github.io/gutenberg/?path=/story/icons-icon--library with the icons you get from checking out this branch and running `npm run storybook:dev`. You can see the new icons at http://localhost:50240/?path=/story/icons-icon--library.

Here are some:

<img width="302" alt="Screenshot 2021-05-06 at 11 56 59" src="https://user-images.githubusercontent.com/1204802/117279671-3b4f1a80-ae62-11eb-9e86-f316a5de89b5.png">

<img width="286" alt="Screenshot 2021-05-06 at 11 57 05" src="https://user-images.githubusercontent.com/1204802/117279687-3db17480-ae62-11eb-884d-e48f1c35d30d.png">

<img width="324" alt="Screenshot 2021-05-06 at 11 59 42" src="https://user-images.githubusercontent.com/1204802/117280199-a567bf80-ae62-11eb-9d18-060b4b6d1d97.png">

I don't love the "cloud upload" icon, but one such is already shipping so I was hoping to find a design to keep the same shape but update it to the newer style.




## Checklist:
